src/FeedForwardPid.cpp

```diff
@@ -17,20 +19,47 @@ FeedForwardPid::FeedForwardPid(double p_gain, double i_gain, double d_gain,
     : p_gain_(p_gain),
       i_gain_(i_gain),
       d_gain_(d_gain),
-      setpoint_(0.0),
+      initialized_(false),
       i_accumulator_(0.0),
       last_current_value_(0.0),
+      setpoint_(0.0),
       i_accumulator_max_(i_accumulator_max),
       i_accumulator_min_(i_accumulator_min)
 {
     // Nothing to do
 }
 
-double FeedForwardPid::update(double current_value, double time_delta)
+bool FeedForwardPid::update(double current_value, double time_delta, double& response)
 {
-    if (time_delta == 0) {
-        time_delta = std::numeric_limits<double>::min();
+    if (!initialized_ && std::isfinite(current_value)) {
```

Comment:
I still don't agree with this, but for the sake of time it's ok. Additionally, I think you should be including the i_term in this since it can be calculated on the first run as well.